.gitignore

```diff
@@ -16,3 +16,5 @@ src/autoscaler/vendor
 workflow
 **/*.iml
 .cache
+scripts/.srl
+privkey.pem
```

Comment:
why is privkey.pem needed here ?
is it a lack of cleanup ?

Reply:
if the generate_test_certs.sh fails due to openssl compatibility it will leave this file hanging around, we wanted to make sure it wasn't commited.